doc/api/test.md

```diff
@@ -1046,6 +1046,9 @@ added:
   - v19.6.0
   - v18.15.0
 changes:
+  - version: v23.0.0
+    pr-url: https://github.com/nodejs/node/pull/54548
+    description: The default reporter is now `spec` unconditionally, even when not in a TTY.
```

Comment:
IMO it would more informative to specify what was the previous default value, for folks trying to support multiple Node.js versions
```suggestion
    description: The default reporter on non-TTY stdout is changed from `tap` to
                 `spec`, aligning with TTY stdout.
```